ansible/ansible.cfg

```diff
@@ -4,5 +4,9 @@ vault_password_file = vault.password
 # Automatically accept SSH thumbprints when machines are provisioned.
 # ATTENTION: DO NOT USE in production environments as it allows for man-in-the-middle attacks
 host_key_checking = False
+remote_user = ec2-user
 
-inventory = ./hosts.yml
+# Tweak some timeout settings to try and prevent ssh failures
```

Comment:
Sorry I haven't made my way in my inbox to your "SSH timeouts" question. I've run into that bunches, too. Things that help:

1. Add an artificial pause after EC2 instances are stood up in your "wait for SSH" task, e.g. the `delay` here: https://github.com/CMSgov/bluebutton-ansible-playbooks-data-sandbox/blob/master/etl_server_redeploy.yml#L98
2. Always, always, always use ansible pipelining:
    * https://github.com/CMSgov/bluebutton-ansible-playbooks-data-sandbox/blob/master/etl_server_redeploy.yml#L112
    * https://github.com/CMSgov/bluebutton-ansible-playbooks-data-sandbox/tree/master/roles/rhel_ansible_pipelining
    * https://github.com/CMSgov/bluebutton-ansible-playbooks-data-sandbox/blob/master/ansible.cfg#L23
3. Bump Ansible's `timeout`: https://github.com/CMSgov/bluebutton-ansible-playbooks-data-sandbox/blob/master/ansible.cfg#L20

Reply:
(I probably made it sound like those aren't complete solutions. They actually are, as far as I can tell: the `-sandbox` plays never seem to run into SSH timeouts anymore for me, with those all in place. I d still have a n intermittent problem with YUM being locked/unavailable after instances start, but it's rare enough that I haven't gone and tracked it down yet.)